src/framed/std/math.clj

```diff
@@ -78,3 +78,28 @@
   (if (> v max-val)
     max-val
     v))
+
+(defn- squared-deviations
+  "Return the set of squared deviations from items in `vs` to their mean"
+  [mean vs]
+  (map #(Math/pow (- % mean) 2) vs))
+
+(defn variance
+  "Compute the variance of `vs`. Divisor used is `n - ddof` where `ddof` represents
+   'delta degrees of freedom'. By default returns an unbiased estimate (ddof = 1)
```

Comment:
Because this concept (both variance and delta degrees of freedom) is used but not explained here, I would recommend linking whatever definition/explanation was used in implementing this.  Some math concepts have multiple definitions and meanings so it's worth it to point to exactly what/why you mean.